packages/client-core/src/components/InviteToast/index.tsx

```diff
@@ -23,65 +23,44 @@ All portions of the code written by the Infinite Reality Engine team are Copyrig
 Infinite Reality Engine. All Rights Reserved.
 */
 
-import React, { useEffect } from 'react'
+import { useEffect } from 'react'
 import { useTranslation } from 'react-i18next'
 
-import { UserName } from '@ir-engine/common/src/schema.type.module'
-import capitalizeFirstLetter from '@ir-engine/common/src/utils/capitalizeFirstLetter'
-import { Button } from '@ir-engine/editor/src/components/inputs/Button'
+import { InviteType, UserName } from '@ir-engine/common/src/schema.type.module'
+
 import { useMutableState } from '@ir-engine/hyperflux'
 
+import { first, isNil } from 'lodash'
+import { NotificationService } from '../../common/services/NotificationService'
 import { InviteService, InviteState } from '../../social/services/InviteService'
 import { AuthState } from '../../user/services/AuthService'
-import styles from './index.module.scss'
 
 const InviteToast = () => {
+  const { t } = useTranslation()
   const inviteState = useMutableState(InviteState)
   const authState = useMutableState(AuthState)
-  const newestInvite =
-    inviteState.receivedInvites.total.value > 0 ? inviteState.receivedInvites.invites[0].value : ({} as any)
-  const { t } = useTranslation()
+  const newestInvite: InviteType = first(inviteState.receivedInvites.invites)?.value as InviteType
 
   useEffect(() => {
     if (inviteState.receivedUpdateNeeded.value && authState.isLoggedIn.value)
       InviteService.retrieveReceivedInvites(undefined, undefined, 'createdAt', 'desc')
   }, [inviteState.receivedUpdateNeeded.value, authState.isLoggedIn.value])
 
-  const acceptInvite = (invite) => {
-    InviteService.acceptInvite(invite)
-  }
-
-  const declineInvite = (invite) => {
-    InviteService.declineInvite(invite)
-  }
-  return (
-    <div
-      className={`${styles.inviteToast} ${
-        inviteState.receivedInvites.total.value > 0 ? styles.animateLeft : styles.fadeOutLeft
-      }`}
-    >
-      <div className={`${styles.toastContainer} `}>
-        {newestInvite?.inviteType && (
-          <span>
-            {capitalizeFirstLetter(newestInvite?.inviteType).replace('-', ' ')} invite from{' '}
-            {newestInvite.user?.name as UserName}
-          </span>
-        )}
-        <div className={`${styles.btnContainer}`}>
-          <Button style={{ color: 'primary' }} className={styles.acceptBtn} onClick={() => acceptInvite(newestInvite)}>
-            {t('social:invite.accept')}
-          </Button>
-          <Button
-            style={{ color: 'primary' }}
-            className={styles.declineBtn}
-            onClick={() => declineInvite(newestInvite)}
-          >
-            {t('social:invite.decline')}
-          </Button>
-        </div>
-      </div>
-    </div>
-  )
+  useEffect(() => {
+    if (!isNil(newestInvite?.inviteType)) {
```

Comment:
`if (newestInvite.inviteType)` should work here

Reply:
done 267035aae831b5dec33d705ec44fb432d4684493